apps/DemoApp/DemoApp/ViewController.m

```diff
@@ -135,20 +135,22 @@ - (AVPlayer *)testAVPlayer {
 - (void)setupAVPlayerViewController:(AVPlayer *)player {
     _avplayer = player;
     _avplayerController.player = _avplayer;
-
+    
```

Comment:
weird whitespace that doesn't matter either way, but since I have a nitpicky review going, might as well keep it going 😬